core/modes/language_modes.py

```diff
@@ -127,7 +51,7 @@ class Actions:
     def code_set_language_mode(language: str):
         """Sets the active language mode, and disables extension matching"""
         global forced_language
-        assert language in language_extensions
+        assert language in code_languages
```

Comment:
I don't believe this works, as code_languages is not a dictionary or list of strings

Reply:
You're completely right. I forgot to update that one.
Fixed now